packages/seacas/libraries/aprepro_lib/apr_builtin.cc

```diff
@@ -1064,6 +1064,7 @@ namespace SEAMS {
         }
       }
       assert(rows - rows_to_skip == (size_t)array_data->rows);
+      delete file;
```

Comment:
Consider using `std::unique_ptr` to manage raw pointers like this.

Reply:
Yes, I definitely need to go through and use many more unique_ptr and shared_ptr...